DQ_test.py

```diff
@@ -107,7 +126,7 @@ def test_ad(self):
     def test_adsharp(self):
         result_of_Adsharp = get_list_of_dq_from_mat('result_of_Adsharp', mat)
         for a, b, c in zip(dq_a_list, dq_b_list, result_of_Adsharp):
-            numpy.testing.assert_almost_equal(vec8(Adsharp(a, b)), vec8(c), 12, "Error in Ad")
+            numpy.testing.assert_almost_equal(vec8(Adsharp(a, b)), vec8(c), 12, "Error in Adsharp")
```

Comment:
@juanjqo  This might be a good time to replace all `12` by `DQ_threshold instead.`